Modules/socketmodule.c

```diff
@@ -8533,6 +8533,9 @@ socket_exec(PyObject *m)
 #ifdef IPV6_RECVDSTOPTS
     ADD_INT_MACRO(m, IPV6_RECVDSTOPTS);
 #endif
+#ifdef  IPV6_RECVERR
```

Comment:
```suggestion
#ifdef IPV6_RECVERR
```

style nit

Reply:
How did that happen? Anyway, thanks, fixed.